src/app-layout/__tests__/main.test.tsx

```diff
@@ -115,6 +115,51 @@ describe.each([
   });
 });
 
+describe.each([
+  [
+    'drawers',
+    {
+      findElement: (wrapper: AppLayoutWrapper) => wrapper.findActiveDrawer(),
+      findToggle: (wrapper: AppLayoutWrapper) => wrapper.findDrawersTriggers()[0],
+      findClose: (wrapper: AppLayoutWrapper) => wrapper.findActiveDrawerCloseButton(),
+    },
+  ],
+] as const)('%s', (name, { findToggle, findElement, findClose }) => {
+  test('property is controlled', () => {
+    const onChange = jest.fn();
+    const drawers = {
+      drawers: {
+        onChange: onChange,
+        items: [
+          {
+            ariaLabels: {
+              closeButton: 'Security close button',
+              content: 'Security drawer content',
+              triggerButton: 'Security trigger button',
+              resizeHandle: 'Security resize handle',
+            },
+            content: <></>,
+            id: 'security',
+            trigger: {
+              iconName: 'security',
+            },
+          },
+        ],
+      },
+    };
+
+    const { wrapper, rerender } = renderComponent(<AppLayout contentType="form" {...drawers} />);
+    findToggle(wrapper).click();
```

Comment:
I recommend to wrap every interaction in `act()`, to ensure side effects from `useEffect` hooks are executed as we expect them to

Reply:
None of the other tests in this file are wrapped in `act()` - should I still do it just for this one?